tanka.hcl

```diff
@@ -0,0 +1,31 @@
+description = "Flexible, reusable and concise configuration for Kubernetes."
+binaries = ["tk"]
+test = "tk --help"
+
+linux {
```

Comment:
FYI if you have two platform blocks that are identical, as these two are, you can just pull them out to the top level.